build/AzurePipelineTemplates/CsWinRT-Build-Steps.yml

```diff
@@ -24,27 +24,14 @@ steps:
 
       Write-Host ##vso[task.setvariable variable=PATH;]${env:Agent_TempDirectory}\procdump;${env:PATH};
 
-# Use .NET Core SDK 2.1
+# Use .NET Core SDK 3.1
 - task: UseDotNet@2
-  displayName: Use .NET Core SDK 2.1
+  displayName: Use .NET Core SDK 3.1
   inputs:
-    version: 2.1.x
+    version: 3.1.x
     installationPath: C:\Users\VssAdministrator\AppData\Local\Microsoft\dotnet\
     performMultiLevelLookup: true
 
-# Install .NET 5 SDK
-- task: PowerShell@2 

```

Comment:
Why don't we need this anymore?

Reply:
The .NET 6 SDK can be used to build .NET 5 TFMs.